rego/rego.go

```diff
@@ -307,6 +317,22 @@ func EvalInterQueryBuiltinCache(c cache.InterQueryCache) EvalOption {
 	}
 }
 
+// EvalRegexBuiltinCache sets the cache that the regex built-in function can utilize
+// during evaluation.
+func EvalRegexBuiltinCache(c cache.ValueCache) EvalOption {
+	return func(e *EvalContext) {
+		e.regexBuiltinCache = c
+	}
+}
+
+// EvalGlobBuiltinCache sets the cache that the glob built-in function can utilize
+// during evaluation.
+func EvalGlobBuiltinCache(c cache.ValueCache) EvalOption {
+	return func(e *EvalContext) {
+		e.regexBuiltinCache = c
```

Comment:
This should set `e.globBuiltinCache`, right?

Reply:
Yes will update.